File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/Arbeidsforholdsgrunnlag.java
package no.nav.domain.pensjon.regler.grunnlag;

import java.io.Serializable;
import java.util.Date;

/**
 * Angir informasjon om stillingsprosent
 * 
 * @author Swiddy de Louw (Capgemini), PK-7973 PKPYTON-526
 */

public class Arbeidsforholdsgrunnlag implements Serializable {



    /**
     * Fom dato for arbeidsforholdet.
     */
    private Date fomDato;

    /**
     * Tom dato for arbeidsforholdet.
     */
    private Date tomDato;

    /**
     * Stillingsandel i prosent.
     */
    private int stillingsprosent;

    /**
     * Navn på arbeidsgiver.
     */
    private String arbeidsgiver;

    /**
     * Arbeidsgivers juridiske organisasjonsnummer.
     */
    private String orgNummer;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/Arbeidsforholdsgrunnlag.kt
package no.nav.pensjon.regler.domain.grunnlag

import java.io.Serializable
import java.util.*
import java.time.LocalDate

class Arbeidsforholdsgrunnlag : Serializable {
    /**
     * Fom dato for arbeidsforholdet.
     */
    var fomDatoLd: LocalDate? = null

    /**
     * Tom dato for arbeidsforholdet.
     */
    var tomDatoLd: LocalDate? = null

    /**
     * Stillingsandel i prosent.
     */
    var stillingsprosent = 0

    /**
     * Navn på arbeidsgiver.
     */
    var arbeidsgiver: String? = null

    /**
     * Arbeidsgivers juridiske organisasjonsnummer.
     */
    var orgNummer: String? = null

}